Packs/CrowdStrikeFalcon/Classifiers/classifier-CrowdStrike_Falcon_Mapper_6.5.json

```diff
@@ -452,9 +621,6 @@
                         ]
                     }
                 },
-                "Behavior": {
-                    "simple": "behaviors"
-                },
```

Comment:
Validation was failing because it couldn't find the incident field **Behavior**, which was added in this [PR](https://github.com/demisto/content/pull/19250), after investigating it with @ssokolovich, we noticed that this incident field does not exist at all. We checked in the content repo, and we did a git log to see if it was present before and got deleted by mistake, but we didn't find anything, therefore, we are deleting it from the mapper.